BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsClient.kt

```diff
@@ -131,6 +137,7 @@ internal class AnalyticsClient @VisibleForTesting constructor(
         } catch (e: Exception) {
             ListenableWorker.Result.failure()
         }
+        }
```

Comment:
Is it just me or does this look weird with two `}` at the same level?

Reply:
Oh yes the indentation was messed up here - fixed!